src/Shared/FileUtilities.cs

```diff
@@ -243,6 +243,58 @@ internal static string EnsureNoTrailingSlash(string path)
             return path;
         }
 
+        /// <summary>
+        /// Ensures the path is enclosed within single quotes.
+        /// </summary>
+        /// <param name="path"></param>
+        /// <returns></returns>
+        internal static string EnsureSingleQuoted(string path)
```

Comment:
I know there was some confusion around names for this. I'll throw out EnsureSingleQuotes/EnsureDoubleQuotes but won't block on it.

Reply:
I'll make the name change. `Quotes` is more consistent with the other methods than 'Quoted'. Thanks